backend/app.py

```diff
@@ -1,5 +1,4 @@
-import sys
```

Comment:
oh yeah is there a reason why we're removing sys here?

Reply:
initially removed since it didn't seem to be needed to run, but since exit() is a sys function, I'm putting import sys back in.